core/trino-main/src/main/java/io/trino/operator/scalar/StringFunctions.java

```diff
@@ -968,4 +969,67 @@ public static Slice soundex(@SqlType(StandardTypes.VARCHAR) Slice slice)
             throw new TrinoException(INVALID_FUNCTION_ARGUMENT, e);
         }
     }
+
+    @Description("Returns Fuzzy Match Simple Ratio")
+    @ScalarFunction("fuzzy_match_simple_ratio")
+    @LiteralParameters({"x", "y"})
+    @SqlType(StandardTypes.BIGINT)
+    public static long fuzzyMatchSimpleRatio(@SqlType("varchar(x)") Slice stringX, @SqlType("varchar(y)") Slice stringY)
```

Comment:
I'd recommend using `left` and `right` naming for the variables instead to be consistent with the notation used within `StringFunctions` class.

See `io.trino.operator.scalar.StringFunctions#levenshteinDistance` for reference.